database/reindex_elastic.js

```diff
@@ -17,13 +16,22 @@ const getIndexUrl = () => {
   return `${elasticUrl}/${config.defaultTenant.indexName}`;
 };
 
+const headers = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+};
+
```

Comment:
As we move forward with replacing libraries with the native Fetch, I would probably put this on an abstracted request object, but much simpler (and less responsibilities) than the one we have now.